packages/stream_chat/lib/src/core/models/filter.dart

```diff
@@ -157,13 +161,20 @@ class Filter extends Equatable {
   factory Filter.exists(String key, {bool exists = true}) =>
       Filter._(operator: FilterOperator.exists, key: key, value: exists);
 
+  /// Matches any list that contains the specified values
+  factory Filter.contains(String key, Object value) =>
+      Filter._(operator: FilterOperator.contains, key: key, value: value);
+
   /// Creates a custom [Filter] if there isn't one already available.
   const factory Filter.custom({
     required Object value,
     String? operator,
     String? key,
   }) = Filter.__;
 
+  /// An empty filter
+  factory Filter.empty() => const Filter.raw(value: {});
```

Comment:
const factory maybe

Reply:
Only redirecting factory constructors can be declared to be `const`